db/blob/blob_file_builder.h

```diff
@@ -87,6 +90,7 @@ class BlobFileBuilder {
   std::unique_ptr<BlobLogWriter> writer_;
   uint64_t blob_count_;
   uint64_t blob_bytes_;
+  std::shared_ptr<IOTracer> io_tracer_;
```

Comment:
Could you please also move this so it's between `write_hint_` and `blob_file_paths_` ?